pkg/apis/configuration/validation/virtualserver.go

```diff
@@ -92,15 +92,19 @@ func (vsv *VirtualServerValidator) validateVirtualServerSpec(spec *v1.VirtualSer
 	return allErrs
 }
 
+const wildcardHost = "*."
+
 func validateHost(host string, fieldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
 	if host == "" {
 		return append(allErrs, field.Required(fieldPath, ""))
 	}
 
-	for _, msg := range validation.IsDNS1123Subdomain(host) {
-		allErrs = append(allErrs, field.Invalid(fieldPath, host, msg))
+	if !strings.HasPrefix(host, wildcardHost) {
```

Comment:
We should also validate the wildcard host - we can use `validation.IsWildcardDNS1123Subdomain`

see [here](https://github.com/kubernetes/kubernetes/blob/v1.24.4/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L250) for reference